.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,30 @@
-<!-- **Ticket:** https://yaits.demos-deutschland.de/Txxyyzz -->
+### Ticket
+https://demoseurope.youtrack.cloud/issue/DPLAN-XXXXX
+or
+https://www.dev.diplanung.de/DefaultCollection/EfA%20DiPlanung/_workitems/edit/XXXXX
```

Comment:
I think that this is too much "internal" information, given that we're inside demosplan-core.

Reply:
But we "always" put those Links, when we develop for projects managed in ADO. Is it more internal, than the YT-link?

Reply:
YT is also automatically linked if we only write the ticket number (DPLAN-XXXX)

Reply:
agreed

Reply:
thanks. Makes it much easier 8077e147ca38c9b66217bd3443c30184d2b98437